bazel/repositories.bzl

```diff
@@ -72,11 +78,25 @@ def _build_recipe_repository_impl(ctxt):
     ctxt.symlink(Label("//ci/prebuilt:BUILD"), "BUILD")
 
     # Run the build script.
-    environment = {}
+    command = []
+    env = {}
+    if ctxt.os.name.upper().startswith("WINDOWS"):
+        vc_path = find_vc_path(ctxt)
+        current_path = get_env_var(ctxt, "PATH", None, False)
+        env = setup_vc_env_vars(ctxt, vc_path)
+        env["PATH"]+=(";%s" % current_path)
+        env["CC"]="cl"
```

Comment:
Can you merge master? We just merged a PR that applied Buildifier on Skylark files that should clean a lot of this up for consistency.